src/demos/calcite-shell-rtl.html

```diff
@@ -574,31 +559,36 @@ <h2 class="heading">My App</h2>
       <p style="text-align: center;">Box Shadow added to show app edges.</p>
       <script>
         var actionPad = document.getElementById("action-pad");
+        var calcitePositionActionPad = document.getElementById("calcite-position-action-pad");
         var actionPadButton = document.getElementById("action-pad-button");
         actionPadButton.addEventListener("click", function() {
           actionPad.hasAttribute("hidden") ? actionPad.removeAttribute("hidden") : actionPad.setAttribute("hidden", "");
-
-          actionPad.positionElement = actionPadButton;
+          calcitePositionActionPad.positionElement = actionPadButton;
+          calcitePositionActionPad.reposition();
         });
 
-        var shellFloatingPanel = document.getElementById("shell-floating-panel");
+        var calcitePositionFloatingPanel = document.getElementById("calcite-position-floating-panel");
+        var shellFloatingPanel = document.getElementById("calcite-shell-floating-panel");
         var shellFloatingPanelButton = document.getElementById("shell-floating-panel-button");
         shellFloatingPanelButton.addEventListener("click", function() {
           shellFloatingPanel.hasAttribute("hidden")
             ? shellFloatingPanel.removeAttribute("hidden")
             : shellFloatingPanel.setAttribute("hidden", "");
 
-          shellFloatingPanel.positionElement = shellFloatingPanelButton;
+          calcitePositionFloatingPanel.positionElement = shellFloatingPanelButton;
```

Comment:
What do you think about supporting an element ID as well as a node? 

Also, if `positionElement` always needs to be set before calling `reposition`, would it be better to update the method signature to include this?

```tsx
calcitePopoverForFloatingPanel.position(elementOrId);
```

And lastly, how does one close the popover ('popunder' 😅)?

Reply:
@jcfranco 

> What do you think about supporting an element ID as well as a node?

I don't think we need it unless you feel strongly about it.

> Also, if positionElement always needs to be set before calling reposition, would it be better to update the method signature to include this?

If you already set the node but want to reposition because of a scrolling div or something than passing the node is unnecessary. We could make it an option argument?

> And lastly, how does one close the popover ('popunder' 😅)?

Just add the `hidden` attribute :) We don't need a method to do that. I can watch the hidden property to stop the events and repositioning. I'll add that.

Reply:
Actually, don't need to watch hidden attribute. Removing referenceElement will stop events.